FeedStoreChallenge/Infrastructure/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Lorenzo Angelini on 18/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+final class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedCache
```

Comment:
`cache` is optional in `FeedStore.xcdatamodel`. Please make it optional here so that the model and the class are consistent.
```suggestion
	@NSManaged var cache: ManagedCache?
```